src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -47,7 +47,7 @@ internal FunctionInvokerBase(ScriptHost host, FunctionMetadata functionMetadata,
 
             PrimaryHostTraceProperties = new Dictionary<string, object>
             {
-                { ScriptConstants.TracePropertyPrimaryHostKey, Metadata.Name }
+                { ScriptConstants.TracePropertyPrimaryHostKey, true }
```

Comment:
This was accidentally changed from "null" I believe to an incorrect string value. Doesn't cause any issues really, since we just rely on existence of the key, but it's best to have it make sense.